plutus-ledger/src/Ledger/Tx.hs

```diff
@@ -269,9 +266,7 @@ To do that we need the required signers which are only available in UnbalancedTx
 So during balancing we can create the SomeCardanoApiTx, while proper validation can only happen in
 Cardano.Node.Emulator.Chain.validateBlock, since that's when we know the right Slot number. This means that
 we need both transaction types in the path from balancing to validateBlock. -}
-data CardanoTx
-    = EmulatorTx { _emulatorTx :: Tx }
-    | CardanoApiTx { _cardanoApiTx :: SomeCardanoApiTx }
+data CardanoTx = CardanoApiTx { _cardanoApiTx :: SomeCardanoApiTx }
```

Comment:
I don't think we need this anymore? `CardanoTx` can be just `SomeCardanoApiTx`.

Reply:
Fair point, I hate to carry around a newtype for the same use but as long as we have this dirty internal, it might be better to do it this way.

Reply:
We could rename `SomeCardanoApiTx` to `CardanoTx`.